pkg/provision/engine.go

```diff
@@ -867,10 +868,44 @@ func (e *NativeEngine) uninstallDeployment(ctx context.Context, dl *gridtypes.De
 
 }
 
+func getZmountSize(wl *gridtypes.WorkloadWithID) (gridtypes.Unit, error) {
```

Comment:
```suggestion
func getZmountSize(wl *gridtypes.Workload) (gridtypes.Unit, error) {
```
you don't need to pass the wrapper type `WorloadWithID`